operator/pkg/reconciliation/constructor.go

```diff
@@ -410,7 +411,13 @@ func buildContainers(dc *api.CassandraDatacenter, serverVolumeMounts []corev1.Vo
 	}
 	loggerContainer.VolumeMounts = []corev1.VolumeMount{cassServerLogsMount}
 
-	return []corev1.Container{cassContainer, loggerContainer}, nil
+	containers := []corev1.Container{cassContainer, loggerContainer}
+	if dc.Spec.Reaper.Enabled {
```

Comment:
should we add that `dc.Spec.ServerType == "cassandra"` also?

Reply:
done